client/install_deployment.go

```diff
@@ -0,0 +1,111 @@
+package client
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strings"
+
+	"github.com/liimaorg/liimactl/client/util"
+)
+
+//CommandOptionsInstallDeployment used for the command options (flags)
+type CommandOptionsInstallDeployment struct {
+	Environment          string `json:"environmentName"`
+	DeploymentDate       string `json:"deploymentDate"`
+	ExecuteShakedownTest bool   `json:"executeShakedownTest"`
+	Wait                 bool   //Wait as long the WaitTime until the deplyoment success or failed
+	MaxWaitTime          int    //Max wait time [seconds] until the deplyoment success or failed
+	FromEnvironment      string //Deploy last deplyoment from given environment
+	Blacklist            string //Blacklist with all appServer, which should not be deployed [filepath]
+}
+
+//Validate the given command options
+func (commandOption *CommandOptionsInstallDeployment) validate() error {
+
+	//Errorlist
+	var errorList []string
+	//Checks and add to errorList if an error
+	util.Check(&errorList, util.ValidateSingleChar(commandOption.Environment), "want environment with one char, got %s", commandOption.Environment)
+	util.Check(&errorList, util.ValidateSingleChar(commandOption.FromEnvironment), "want FromEnvironment with one char, got %s", commandOption.FromEnvironment)
+
+	//Return all errors as one
+	if len(errorList) > 0 {
+		return errors.New(strings.Join(errorList, ", "))
+	}
+	return nil
+}
+
+//InstallDeployment create multiple deployments and returns the deploymentresponse

```

Comment:
create -> creates

Reply:
Better call it PromoteDeployments, rename install_deployment.go and CommandOptionsInstallDeployment to